Add sentiment analysis to feedback form with modal UI

linaya2004 opened the pull request:
- Integrated TextBlob for sentiment analysis
- Refactored feedback view to use query parameters instead of template rendering
- Implemented Bootstrap modal for thank-you message
- Styled modal to match Hardhat theme and added logo


https://github.com/user-attachments/assets/7c65f284-e12f-4c94-ba7c-ce23729851e5

vaqwerty2 commented:
Works well, but need to add TextBlob to the requirements.txt.

vaqwerty2 reviewed (changes requested):
Please add TextBlob to the requirements.txt.

linaya2004 commented:
> Works well, but need to add TextBlob to the requirements.txt.



> Please add TextBlob to the requirements.txt.
Hi, will do.
Also a quick heads up—I'm using my friend's computer to work and push the code because my laptop is low on space. That's why the commit history may show a different author name even though it is my code.

linaya2004 commented:
hello i have added TextBlob to the requirements.txt file as requested.

Also, as part of this update, I implemented a complete 5-star rating system with the following features:

Sentiment analysis using TextBlob

Confetti animation triggered for 5-star feedback

Dynamic display of the average rating

Live feedback labels such as “Very Good” or “Poor”

Visual and layout enhancements for the thank-you page.
